tests/unit/pushnotifications/test_pushservice.py

```diff
@@ -84,9 +95,11 @@ def test_send_on_blockchain_event(raw_push_service, recorder, web3_event_transfe
 
 
 def test_send_on_non_blockchain_event(raw_push_service, recorder):
-    raw_push_service.send_event(client_token="token", event=message_event)
+    raw_push_service.send_event(
+        client_token="token", event=payment_request_message_event
+    )
 
-    assert_body_has_correct_payload(recorder, message_event)
+    assert_body_has_correct_payload(recorder, payment_request_message_event)
```

Comment:
You could parametrize this test to also run with the decline message event

Reply:
ok will do

Reply:
done